features/step_definitions/logout_steps.rb

```diff
@@ -1 +1,17 @@
-#Add your codez here
+Given(/^the secret page$/) do
+  within("#session") do
```

Comment:
Probs not, but wasn't sure if it was necessary in this challenge, but definitely better practice.